api/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/api/lagreselvbestemtim/LagreAapenImRoute.kt

```diff
@@ -34,28 +34,28 @@ import no.nav.helsearbeidsgiver.utils.pipe.orDefault
 import java.util.UUID
 
 // TODO test
-fun Route.lagreAapenImRoute(
+fun Route.lagreSelvbestemtImRoute(
     rapid: RapidsConnection,
     tilgangskontroll: Tilgangskontroll,
     redisPoller: RedisPoller
 ) {
-    val producer = LagreAapenImProducer(rapid)
+    val producer = LagreSelvbestemtImProducer(rapid)
 
-    post(Routes.AAPEN_INNTEKTMELDING_MED_VALGFRI_ID) {
-        val aapenIdFraPath = call.parameters["aapenId"]
+    post(Routes.SELVBESTEMT_INNTEKTMELDING_MED_VALGFRI_ID) {
+        val selvbestemtIdFraPath = call.parameters["selvbestemtId"]
```

Comment:
Jeg er litt usikker på om selvbestemtId bør eksponeres for sluttbruker.. Kanskje bare id? 
Gjelder også Route / Path, kanskje "ny-inntektsmelding" eller lignende er mer nøytralt å eksponere ut til hele verden. 
Disse tingene må jo uansett også synkes med frontend når man endrer, men det regner jeg med at du har kontroll på / husker.

Reply:
Jeg er enig! Disse verdiene skal endres når vi spikrer det skikkelig sammen med frontend, men enn så lenge så har jeg latt det beholde backenden sin "terminologi".